tests/components/component-playground/selected-fixture-and-editor/render/dom.js

```diff
@@ -45,4 +45,8 @@ describe(`ComponentPlayground (${FIXTURE}) Render DOM`, function() {
     expect($(component.refs.editor.getDOMNode())
            .hasClass(style['invalid-syntax'])).to.be.true;
   });
+
+  it('should not render a splitpane', function() {
+    expect(component.refs.editorPreviewSplitPane).to.exist;
```

Comment:
please test all the props that you're sending to `react-split-pane` component. It's not sufficient just to ensure that the DOMNode exists.